docs/release-notes/8.15.asciidoc

```diff
@@ -21,6 +21,18 @@ On August 1, 2024, it was discovered that Elastic AI Assistant's responses when
 ====
 // end::known-issue-189676[]
 
+// tag::known-issue-5713[]
+[discrete]
+.The option to manually run multiple rules is available in the bulk actions menu on the Rules page
+[%collapsible]
+====
+*Details* +
+On August 20, 2024, it was discovered that the bulk actions menu on the Rules page erroneously had the option to manually run multiple rules.  
+
+*Workaround* +
+To resolve this issue, upgrade to 8.15.1 or later.
```

Comment:
is it ok, if 8.15.1 not yet available?

Reply:
If you'd prefer to wait til 8.15.1 is available, I can remove this statement for now and then re-add it later.

Reply:
Removed in [52a225c](https://github.com/elastic/security-docs/pull/5714/commits/52a225c9a072f1c813a43bfe014e10c69849678b).